### PR TITLE
Feature/most popular section

### PR DESCRIPTION
**Pull request by vinayak-4**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+notes.txt
```
Comment (Delwalt) [inline .gitignore:2]: notes.txt ???

File: script.js [hunk 2]
```diff
@@ -9,3 +10,82 @@ mobileNavOpen.addEventListener("click", () => {
...
+      "X-RapidAPI-Host": "rawg-video-games-database.p.rapidapi.com",
+    },
+  };
+  try {
+    const response = await fetch(
+      "https://rawg-video-games-database.p.rapidapi.com/games?key=cf5e35876a3546608b18bd8bc95faa6a&page_size=40&ordering=-rating",
+      options
+    );
+    const data = await response.json();
+    console.log(data.results);
```
Comment (Delwalt) [inline script.js]: remove console

File: script.js [hunk 3]
```diff
@@ -9,3 +10,82 @@ mobileNavOpen.addEventListener("click", () => {
...
+    addItemToList(data.results);
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+function createElementWithClass(tagName, ...className) {
+  // console.log(...className);
```
Comment (Delwalt) [inline script.js]: remove comment

File: script.js [hunk 4]
```diff
@@ -9,3 +10,82 @@ mobileNavOpen.addEventListener("click", () => {
...
+function createElementWithId(tagName, idName) {
+  const element = document.createElement(tagName);
+  element.setAttribute("id", idName);
+  return element;
+}
+
+function addItemToList(data) {
+  const gameData = data.slice(0, 8);
+
+  console.log("gameList", data);
```
Comment (Delwalt) [inline script.js]: remove console

File: script.js [hunk 5]
```diff
@@ -9,3 +10,82 @@ mobileNavOpen.addEventListener("click", () => {
...
+  }
+  return element;
+}
+function createElementWithId(tagName, idName) {
+  const element = document.createElement(tagName);
+  element.setAttribute("id", idName);
+  return element;
+}
+
+function addItemToList(data) {
```
Comment (Delwalt) [inline script.js]: give it a better name

**Review by Delwalt — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (vinayak-4) [reply .gitignore:2]: it is a temporary file that has notes related to the project or data that might be required in near future.

File: script.js (hunk 2 above)
Comment (vinayak-4) [reply script.js]: okay I'll remove it

File: script.js (hunk 3 above)
Comment (vinayak-4) [reply script.js]: Okay I'll remove it

**Review by Delwalt — APPROVED** [review]

**Pull request merged**